tests/regression

```diff
@@ -135,8 +144,7 @@ setup() {
 }
 
 teardown() {
-    kill "$(cat idxr_pid)" 2>/dev/null || true
-    rm -f ./idxr_pid
+    kill_idxr || rm -f ./idxr_pid
```

Comment:
If the above suggestion (for `kill_idxr`) is implemented, then I think this line can be:

 ```suggestion
     kill_idxr
```

Reply:
This is an attempt to guard against a future version of 'kill_idxr' which could conceivably return an error. In that case, we still proceed to remove the PID file. You only write the line without that feature because you have knowledge of the implementation - namely, that it will never return if an error is encountered.